care/facility/models/patient_tele_consultation.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+from multiselectfield import MultiSelectField
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'multiselectfield'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5331745)